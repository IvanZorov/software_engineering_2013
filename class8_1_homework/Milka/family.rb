require 'csv'

# vavejdash marka i cena i ti dava sichko koeto mojesh da si kypish

array=Array.new()
product=ARGV[1]
price=ARGV[2].to_i
CSV.foreach(ARGV[0]) do |row|
	if (row[0] == product && row [2].to_i <= price)
	array << [row[1],row[2],row[3]]
	end
end
array.sort!{|a,b| a[1].to_i <=> b[1].to_i}
CSV.open("family_results.csv" , "w") do |csv|
	array.each do |element|
		csv << element
	end
end
