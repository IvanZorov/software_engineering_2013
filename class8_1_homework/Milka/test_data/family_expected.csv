tracksuit,56.00,M
tracksuit,71.00,W
sneakers,92.00,W
sweatshirt,110.00,W
sneakers,116.00,M
sweatshirt,120.00,M
